aws_s3_policies/aws_s3_bucket_secure_access.py

```diff
@@ -3,17 +3,51 @@
 from panther_base_helpers import deep_get
 from policyuniverse.policy import Policy
 
+# According to AWS there should exist an explicit deny policy that contains
+# "aws:SecureTransport": "false" for this check to be compliant
+# https://aws.amazon.com/premiumsupport/knowledge-center/s3-bucket-policy-for-config-rule/
+# This policy returns a LOW severity alert if the bucket policy is using an implicit deny
+# of secure transport or a HIGH severity if secure transport is not enforced.
+
+EXPLICIT_DENY = False
+IMPLICIT_DENY = True
+
 
 def policy(resource):
     if resource["Policy"] is None:
         return False
 
+    global EXPLICIT_DENY, IMPLICIT_DENY  # pylint: disable=global-statement
```

Comment:
We may need to reset these to their default values on each run of the policy.  See here: https://github.com/panther-labs/panther-analysis/pull/278#discussion_r654716402 

Also, I think `EXPLICIT_DENY` probably doesn't need to be a global variable since we don't use it anywhere else